llvm/lib/Transforms/Utils/LCSSA.cpp

```diff
@@ -317,6 +316,23 @@ bool llvm::formLCSSAForInstructions(SmallVectorImpl<Instruction *> &Worklist,
   return Changed;
 }
 
+/// For every instruction from the worklist, check to see if it has any uses
+/// that are outside the current loop.  If so, insert LCSSA PHI nodes and
+/// rewrite the uses.
+bool llvm::formLCSSAForInstructions(SmallVectorImpl<Instruction *> &Worklist,
+                                    const DominatorTree &DT, const LoopInfo &LI,
+                                    ScalarEvolution *SE,
+                                    SmallVectorImpl<PHINode *> *PHIsToRemove,
+                                    SmallVectorImpl<PHINode *> *InsertedPHIs) {
+  // Cache the Loop ExitBlocks computed during the analysis.  We expect to get a
+  // lot of instructions within the same loops, computing the exit blocks is
+  // expensive, and we're not mutating the loop structure.
```

Comment:
I'd suggest moving this comment to the definition of LoopExitBlocksTy. No need to repeat it three times.

Reply:
done